foundation/foundation-upgrade/src/test/resources/test-files/v1_7_0/DisablePythonLinting/migration/add-linting/pom.xml

```diff
@@ -0,0 +1,180 @@
+    <maven.snapshot.repo.id>maven-snapshots</maven.snapshot.repo.id>
+    <maven.snapshot.repo.url>https://snapshot-PLACEHOLDER/repository/maven-snapshots</maven.snapshot.repo.url>
+    <pypi.project.repository.url>https://pypi-PLACEHOLDER/repository/</pypi.project.repository.url>
+  </properties>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.boozallen.aissemble</groupId>
+        <artifactId>bom-component</artifactId>
+        <version>${version.aissemble}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.boozallen.aissemble</groupId>
+        <artifactId>bom-instantiation</artifactId>
+        <version>${version.aissemble}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+  <scm>
+    <connection>scm:git:url.git</connection>
+    <developerConnection>scm:git:url.git</developerConnection>
+    <url>url</url>
+    <tag>HEAD</tag>
+  </scm>
+  <distributionManagement>
+    <!-- The repositories that the maven artifacts will be deployed
+		to. The credentials for the repository must be provided in the Maven
+		settings.xml file. The repository/snapshotRepository id below should
+		match the serverId in the settings.xml -->
+    <repository>
+      <id>${maven.repo.id}</id>
+      <url>${maven.repo.url}</url>
+    </repository>
+    <snapshotRepository>
+      <id>${maven.snapshot.repo.id}</id>
+      <url>${maven.snapshot.repo.url}</url>
+    </snapshotRepository>
+  </distributionManagement>
+  <repositories>
+    <repository>
+      <id>${maven.repo.id}</id>
+      <url>${maven.repo.url}</url>
+      <releases>
+        <enabled>true</enabled>
+        <checksumPolicy>fail</checksumPolicy>
+      </releases>
+    </repository>
+    <repository>
+      <id>${maven.snapshot.repo.id}</id>
+      <url>${maven.snapshot.repo.url}</url>
+      <snapshots>
+        <enabled>true</enabled>
+        <checksumPolicy>fail</checksumPolicy>
+      </snapshots>
+    </repository>
+  </repositories>
+  <build>
+    <extensions>
+      <extension>
+        <groupId>com.boozallen.aissemble</groupId>
+        <artifactId>foundation-mda</artifactId>
+        <version>${version.aissemble}</version>
+      </extension>
+    </extensions>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.technologybrewery.fermenter</groupId>
+          <artifactId>fermenter-mda</artifactId>
+          <dependencies>
+            <dependency>
+              <groupId>com.boozallen.aissemble</groupId>
+              <artifactId>foundation-mda</artifactId>
+              <version>${version.aissemble}</version>
+            </dependency>
+          </dependencies>
+        </plugin>
+        <plugin>
+          <groupId>org.technologybrewery.habushu</groupId>
+          <artifactId>habushu-maven-plugin</artifactId>
+          <configuration>
```

Comment:
A: Might add some configuration so it's a bit more realistic. Not many people will have an empty config block.

Reply:
Added the skipDeploy configuration for a more realistic test. Thanks Emily!